tests/integration/targets/podman_image/tasks/main.yml

```diff
@@ -176,6 +176,23 @@
         path: /var/tmp/build
       register: oci_build2
 
+    - name: Build OCI image from a directory without Containerfile (should fail)
+      containers.podman.podman_image:
+        executable: "{{ test_executable | default('podman') }}"
+        name: testimage
```

Comment:
it should be a different image name, since `testimage` already exists and it's not rebuilt